submissions/MosQuitO404/js-dom/index.html

```diff
@@ -0,0 +1,87 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>DOM-F1</title>
+</head>
+<body>
+    <header class="header-container">
+        <span class="header-logo-wrapper"><img class="logo-img" src="teams_img/F1-icon-square.png" alt=""></span>
+        <span class="header-title"><h1>Racing Teams 2022</h1></span>
+        <div class="hamb-wrapper">
+           <div class="hamb">
+                <div class="hamb-field" id="hamb">
+                    <span class="bar"></span>
+                </div>
+           </div>
+        </div>
+    </header>
+    <div class="popup" id="popup"></div>
+    <menu class="menu-container" id="menu">
+        <li class="menu-team" id="alfa-romeo" tabindex="0" data-action="alfa-romeo">
```

Comment:
You should have only one ```data-action``` per list item. Don't add data-action to each child of the list item. Instead of this find the list item using ```parentNode``` and cycle.

Reply:
Done!